package.json

```diff
@@ -48,30 +48,30 @@
 		"babel-preset-env": "^1.7.0",
 		"babel-preset-stage-2": "^6.24.1",
 		"chai": "^4.2.0",
-		"css-loader": "^2.1.1",
-		"eslint": "^4.18.1",
-		"eslint-config-airbnb-base": "^12.1.0",
-		"eslint-config-standard": "^11.0.0-beta.0",
-		"eslint-plugin-import": "^2.18.2",
-		"eslint-plugin-node": "^5.2.1",
-		"eslint-plugin-promise": "^3.8.0",
-		"eslint-plugin-standard": "^3.0.1",
-		"file-loader": "^2.0.0",
+		"css-loader": "^3.4.2",
+		"eslint": "^6.8.0",
+		"eslint-config-airbnb-base": "^14.0.0",
+		"eslint-config-standard": "^14.1.0",
+		"eslint-plugin-import": "^2.20.1",
+		"eslint-plugin-node": "^11.0.0",
+		"eslint-plugin-promise": "^4.2.1",
+		"eslint-plugin-standard": "^4.0.1",
+		"file-loader": "^5.1.0",
 		"html-webpack-plugin": "^3.2.0",
 		"istanbul": "^0.4.5",
-		"jest": "^23.6.0",
-		"jsdom": "^13.2.0",
-		"mocha": "^5.2.0",
-		"sinon": "^4.2.2",
-		"sinon-chai": "^2.14.0",
-		"style-loader": "^0.23.1",
-		"webpack": "^4.39.2",
-		"webpack-cli": "^3.3.7",
-		"webpack-dev-server": "^3.8.0",
+		"jest": "^25.1.0",
+		"jsdom": "^16.2.0",
+		"mocha": "^7.1.0",
+		"sinon": "^9.0.0",
+		"sinon-chai": "^3.5.0",
+		"style-loader": "^1.1.3",
+		"webpack": "^4.42.0",
+		"webpack-cli": "^3.3.11",
+		"webpack-dev-server": "^3.10.3",
 		"webpack-uglify-js-plugin": "^1.1.9"
 	},
 	"dependencies": {
-		"eventemitter3": "^3.1.0",
```

Comment:
Wondering if we can break something with this new version

Reply:
I tried the "npm run test" and "npm start" and both seem to be working as intended.